source/style-guide/style/links-and-cross-references.txt

```diff
@@ -4,39 +4,56 @@
 Links and Cross-References
 ==========================
 
-.. include:: /style-guide/includes/page-needs-update.rst
+.. default-domain:: mongodb
+
+.. include:: /includes/styles/corrections.rst
 
 Use cross-references to help users navigate content and find content
 that's related to what they're currently viewing. Cross-references can
 be linked or not linked, depending on the location of the content to
 which you're referring.
 
+.. note::
+
+   {+rst+} refers to links between arbitrary sections between documents
+   or entire documents using special roles as
+   :sphinx-rst:`cross references <roles.html#cross-referencing-syntax>`
+   and links to other URLs as
+   :sphinx-rst:`links <basics.html#hyperlinks>`. This document refers to both as cross references for expediency.
+
 -  When you refer to content within the same article or section, such
    as tables, figures, examples, or a subsection, create a simple
-   textual cross-reference that isn't linked.
-
-   Users typically expect links to take them to a location outside of
-   the article or section that they're currently reading, so links that
-   just jump to another place in the same article or section can be
-   confusing. Exceptions are a TOC, or *jump list*, at the top of an
-   article or section that provides links to the high-level headings in
-   the article or section, and "back to top" links that take the user
-   back to the top of the page.
+   textual cross-reference. If the referenced copy falls much further
+   down the page, link to that content.
 
 -  When you refer to other content, whether created by MongoDB or
-   outside of MongoDB, provide a link to that content. Ensure that the
-   link is active and that the content is up-to-date. Periodically
-   check the link and content.
+   outside of MongoDB, provide a link to that content. Verify that the
+   link works and that the content current. Periodically check the link
+   and content. If you link to Wikipedia or any other site that uses
+   history, link to a specific revision of that page or content.
+
+   .. example::
+
+      .. code-block:: rst
+
+         Timestamp in the number of seconds that have elapsed since the
+         :wikipedia:`UNIX epoch </Unix_time?oldid=828172017>` when the
+         someone last updated this setting.
+
+.. _sg-link-text:
+
+Cross Reference and Link Text
```

Comment:
In the admonition above, it says: "This document refers to both [cross references and links] as cross references for expediency."

But in this header, and throughout the rest of the document, the phrases "cross reference and link" and ""cross references and links" appear. For consistency, I think either remove "and link(s)" everywhere, or remove the admonition.

Reply:
Updated. It didn't read correctly otherwise.